Example/Podfile

```diff
@@ -0,0 +1,19 @@
+# Uncomment the next line to define a global platform for your project
```

Comment:
delete this podfile because it is unnecessary (also remove the associated workspace)